pylintrc

```diff
@@ -50,7 +50,7 @@ output-format=text
 files-output=no
 
 # Tells whether to display a full report or only the messages
-reports=yes
+reports=no
```

Comment:
I dislike this report too but we should add more tests for it in pylint if we remove it from astroid (we already broke it and realized only because it was used in astroid)

Reply:
On the other hand, I never look at the report anyway.